### PR TITLE
Fix for the Jira Issue - EMM-979

### PR DESCRIPTION
**Pull request by Kamidu**

### CODE REVIEW

File: modules/apps/jaggery/mdm/src/mdm/modules/policy.js [hunk 1]
```diff
@@ -85,9 +85,14 @@ policyModule = function () {
                 response.status = "success";
                 response.content = policyListToView;
                 return response;
-            },function(){
+            },function(responsePayload){
                 var response = {};
                 response.status = "error";
+                if(responsePayload  == "Scope validation failed"){
```
Comment (harshanL) [inline modules/apps/jaggery/mdm/src/mdm/modules/policy.js]: Check formatting of if blocks.

File: modules/apps/jaggery/mdm/src/mdm/modules/policy.js [hunk 2]
```diff
@@ -85,9 +85,14 @@ policyModule = function () {
                 response.status = "success";
                 response.content = policyListToView;
                 return response;
-            },function(){
+            },function(responsePayload){
```
Comment (kasundananjaya) [inline modules/apps/jaggery/mdm/src/mdm/modules/policy.js]: Fix formatting issues.

Comment (kasundananjaya) [inline modules/apps/jaggery/mdm/src/mdm/modules/policy.js]: Fix formatting issues.

Comment (Kamidu) [inline modules/apps/jaggery/mdm/src/mdm/modules/policy.js]: These formatting issues has been fixed in a recent PR

File: modules/apps/jaggery/mdm/src/mdm/modules/policy.js (hunk 1 above)
Comment (Kamidu) [inline modules/apps/jaggery/mdm/src/mdm/modules/policy.js]: These formatting issues has been fixed in a recent PR

**Pull request merged**